index.js

```diff
@@ -74,7 +74,18 @@ function magnetURIDecode (uri) {
       }
     })
   }
+
+  if (result.xs) {
+    const xss = Array.isArray(result.xs) ? result.xs : [result.xs]
+    xss.forEach(xs => {
+      if ((m = xs.match(/^urn:btpk:(.{64})/))) {
+        result.publicKey = m[1]
```

Comment:
This is a hex string, right? Why no `.toLowerCase()` like we did for `xt` above?